src/main/java/com/blamejared/contenttweaker/wrappers/MCItemUseContext.java

```diff
@@ -0,0 +1,62 @@
+package com.blamejared.contenttweaker.wrappers;
+
+import com.blamejared.crafttweaker.api.annotations.ZenRegister;
+import com.blamejared.crafttweaker.impl.item.MCItemStackMutable;
+import com.blamejared.crafttweaker.impl.util.MCDirection;
+import com.blamejared.crafttweaker_annotations.annotations.Document;
+import com.blamejared.crafttweaker_annotations.annotations.ZenWrapper;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemUseContext;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.openzen.zencode.java.ZenCodeType;
+
+/**
+ * A class to handle some data when using item.
+ */
+@ZenRegister
+@ZenCodeType.Name("mods.contenttweaker.item.MCItemUseContext")
+@Document("mods/contenttweaker/API/item/MCItemUseContext")
+@ZenWrapper(wrappedClass = "net.minecraft.item.ItemUseContext")
+public class MCItemUseContext {
+    private final ItemUseContext internal;
+
+    public MCItemUseContext(ItemUseContext internal) {
+        this.internal = internal;
+    }
+
+    @ZenCodeType.Getter("player")
+    @ZenCodeType.Nullable
+    public PlayerEntity getPlayer() {
+        return internal.getPlayer();
+    }
+
+    @ZenCodeType.Getter("pos")
+    public BlockPos getPos() {
+        return internal.getPos();
+    }
+
+    @ZenCodeType.Getter("face")
+    public MCDirection getDirection() {
```

Comment:
Any reason why method Name and Getter name are different?
Like, why not `getFace()` , or `@Getter("direction")`